slither/visitors/slithir/expression_to_slithir.py

```diff
@@ -523,6 +536,10 @@ def _post_member_access(self, expression: MemberAccess) -> None:
             if expression.member_name in expr.custom_errors_as_dict:
                 set_val(expression, expr.custom_errors_as_dict[expression.member_name])
                 return
+            # Lookup enums when in a different contract e.g. C.E
+            if str(expression) in expr.enums_as_dict:
```

Comment:
Can this be `expression.member_name` instead of `str(expression)`?

Reply:
`expression.member_name` has only the enum name e.g. with C.E it's E but the fully qualified name is needed and that's the `expression`